project/Dependencies.scala

```diff
@@ -10,10 +10,6 @@ object Dependencies {
       */
     val crossScalaVersions = Seq(scala213, scala212)
 
-    /** Fuller cross-versions (used for injected packages like stryker4s-api and sbt-stryker4s-testrunner)
-      */
-    val fullCrossScalaVersions = crossScalaVersions ++ Seq(scala3)
```

Comment:
Why was this removed? The injected sbt testrunner still needs to be published for the fullest range possible (so that includes Scala 3)

Reply:
Is there a use-case for using fullCrossScalaVersions for api and testrunner? The sbt plugin which uses testrunner is only built with scala 2_12 and all other modules are built with 2_12 or 2_13.

Reply:
Yes, api and testrunner are injected libraries when starting a testrunner. It runs alongside the tests. So they need to be built on whatever scala version the end-users project is, otherwise they'll get conflicts of mismatched Scala versions